Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -10,6 +10,10 @@ module.exports.Quantity = class Quantity extends Expression
 
   exec: (ctx) ->
     @
+  
+  toString: () ->
+    "#{@value}#{@unit}"
+  
```

Comment:
Does this match the spec?  According to page 161, the string representation of a quantity is: `(-)?#0.0# '<unit>'`.